templates/CRM/Contribute/Form/Contribution.tpl

```diff
@@ -246,7 +240,7 @@
           {ts}Payment Details{/ts}
         </div>
         <div class="crm-accordion-body">
-        {if $contribID && $payments}
+        {if $contribID && ($payments || $isPending)}
```

Comment:
as $isPending is ONLY set if contribution status is ```Pending``` so I thought why not name it so to add meaning